DESCRIPTION

```diff
@@ -90,9 +90,13 @@ Suggests:
     ggplot2,
     reactlog (>= 1.0.0),
     magrittr,
+    shinytest,
     yaml,
     future,
     dygraphs
+Remotes:
```

Comment:
These Remotes aren't needed, right?

Reply:
I have no idea where these are coming from. When I git blame it's leading me to https://github.com/rstudio/shiny/commit/3b04c642aeaeb3da4159f823983f07fb173c78b1 ?

 I guess I'll just delete them.